bench/Sender/Main.hs

```diff
@@ -56,7 +57,7 @@ main = do
     loadLogConfig logsPrefix logConfig
     setLocaleEncoding utf8
 
-    Right transport_ <- TCP.createTransport "0.0.0.0" "3432" TCP.defaultTCPParameters
+    Right transport_ <- TCP.createTransport "0.0.0.0" "127.0.0.1" "3432" TCP.defaultTCPParameters
```

Comment:
Oh, I see. Took me a while to realize what the issue is. `address :: EndPoint -> EndPointAddress` will be `0.0.0.0:<port>:<id>` if we bind to `0.0.0.0`, but of course we don't want to tell our peers to connect to this address.